lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -299,6 +307,24 @@
   stampTemplate('unoptimized-images-tmpl', document.body);
 }
 
+function deprecationsTest() {
+  stampTemplate('deprecations-tmpl', document.head);
+
+  // Some APIs atm are not triggering Chrome's deprecation messags. https://crbug.com/680832.
+  // try {
+  //   MediaStreamTrack.getSources(); // FAIL
+  // } catch(e) {}
+
+  // console.timeline(); // FAIL
```

Comment:
would it be crazy to uncomment these so we'd be forced to keep up to date as they're fixed and so the expectations file would no longer match? :)